lib/absinthe/schema/notation.ex

```diff
@@ -423,6 +423,21 @@ defmodule Absinthe.Schema.Notation do
     :ok
   end
 
+  @placement {:complexity, [under: [:field]]}
+  defmacro complexity(func_ast) do
```

Comment:
I am always impressed when someone manages to modify our insane Notation macros. Bravo 😉